Models/SSBTaxAssessment/POCO/Dimension.cs

```diff
@@ -0,0 +1,108 @@
+using System.Collections.ObjectModel;
+using System.Text.Json.Serialization;
+
+namespace Backend.Models.SSBTaxAssessment.POCO
+{
+    public class Dimension
+    {
+        [JsonPropertyName("Region")] public Region Region { get; set; }
+
+        [JsonPropertyName("Alder")] public Alder Alder { get; set; }
```

Comment:
Mix of Norwegian and English here. Is it possible to keep the variable names in English, and rather let the `JsonPropertyName`s (which I assume have to match the incoming JSON) in Norwegian?